### PR TITLE
[SLT-45] Bump to php 8

### PR DESCRIPTION
**Pull request by orakili**
Ticket: SLT-45

Extend https://github.com/UN-OCHA/slt8-site/pull/37 with also a bump to PHP 8.

Tested the site and it's working fine.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build the code.
-FROM unocha/unified-builder:${TAG:-7.4-develop} as builder
+FROM public.ecr.aws/unocha/unified-builder:8.0-experimental as builder
```
Comment (cafuego) [inline docker/Dockerfile:2]: This flips it from the DocherHub registry to the AWS one. I think that's fine, though generally I'd like to swap them over once the *site* moves to AWS as well.

File: symfony.lock [hunk 2]
```diff
@@ -222,17 +225,14 @@
         "version": "1.0.0-alpha7"
     },
     "drush/drush": {
-        "version": "10.5.0"
+        "version": "10.3.6"
```
Comment (cafuego) [inline symfony.lock:228]: Downgrade?

Comment (orakili) [reply symfony.lock:228]: @cafuego Not really my doing, when the php requirement is set to PHP 8.0, composer downgrades drush...

File: docker/Dockerfile (hunk 1 above)
Comment (orakili) [reply docker/Dockerfile:2]: @cafuego Do you mean having the *site* images on AWS?

Comment (cafuego) [reply docker/Dockerfile:2]: Yeah, we'll eventually stop using dockerhub. RW9 is already in the AWS registry and not on dockerhub.

**Pull request closed without merging**